Headers/DebugServer2/Utils/Log.h

```diff
@@ -34,7 +34,8 @@ enum {
 uint32_t GetLogLevel();
 void SetLogLevel(uint32_t level);
 void SetLogColorsEnabled(bool enabled);
-void SetLogOutputStream(FILE *stream);
+const char *GetLogOutputFilename();
+void SetLogOutputFilename(const char *filename);
```

Comment:
I think you should change these APIs to use `std::string()` instead of `char *`. It'll make the implementation simpler. It'll also conveniently allow you to ignore this `PATH_MAX` thing.